kvbc/include/Replica.h

```diff
@@ -93,6 +93,12 @@ class Replica : public IReplica,
 
   // Get the last block ID in the system.
   BlockId getLastBlockId() const override final;
+
+  // Get the value of 'key' in `category_id` as it is in 'snapshot_version'
```

Comment:
"Get the version of `key` that is the closest from below or equal to `snapshot_version`"

Reply:
ok, thanks